src/nv_ingest/util/image_processing/table_and_chart.py

```diff
@@ -38,13 +41,26 @@ def join_cached_and_deplot_output(cached_text, deplot_text):
     """
     chart_content = ""
 
-    if (cached_text is not None) and (deplot_text is not None):
+    if (cached_text is not None):
```

Comment:
The order should strictly be: chart_title from cached, then deplot output, then other cached fields. We have experimented with different ordering, and found this to be the best. i.e, deplot output should not be at the very end.

Also, `caption` is not a cached field. It was from the captioning model (no longer used).

So it should be something like 
```
    if (cached_text is not None):
        cached_text_dict = json.loads(cached_text)

    chart_content += cached_text_dict.get("chart_title", "")

    if (deplot_text is not None):
        chart_content += f" {deplot_text}"

    chart_content += " " + cached_text_dict.get("x_title", "")
    chart_content += " " + cached_text_dict.get("xlabel", "")
    chart_content += " " + cached_text_dict.get("y_title", "")
    chart_content += " " + cached_text_dict.get("ylabel", "")
    chart_content += " " + cached_text_dict.get("legend_label", "")
    chart_content += " " + cached_text_dict.get("legend_title", "")
    chart_content += " " + cached_text_dict.get("mark_label", "")
    chart_content += " " + cached_text_dict.get("value_label", "")
    chart_content += " " + cached_text_dict.get("other", "")
```
why do we need `except json.JSONDecodeError:`? Have you seen this error?

Reply:
> why do we need except json.JSONDecodeError:? Have you seen this error?

That was in the original code that this was created from.I took it as an indication that sometimes we would not get JSON back.